AppCenter/AppCenter/Internals/Util/MSJwtClaims.h

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import <Foundation/Foundation.h>
+
+@interface MSJwtClaims : NSObject
+
+/**
+ * The JWT subject.
+ */
+@property(nonatomic, copy, readonly) NSString *subject;
+
+/**
+ * The JWT expiration date.
+ */
+@property(nonatomic, copy, readonly) NSDate *expirationDate;
+
+/**
+ * Parses the JWT.
+ *
+ * @param jwt The jwt string.
+ *
+ * @return MSJwtCLaims instance if the string is a valid JWT.
+ */
++ (MSJwtClaims *)parse:(NSString *)jwt;
+
+/**
+ * Returns the subject.
+ *
+ * @return NSString The subject.
+ */
+- (NSString *)getSubject;
```

Comment:
Objective-C naming convention doesn't use `get` prefix in the getter method.

Because `subject` is already a readonly property, you don't need this getter method. You can call the `subject` property directly.